src/createServer.js

```diff
@@ -1,3 +1,60 @@
+/* eslint-disable max-len */
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    try {
+      const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+      const textToConvert = normalizedURL.pathname.slice(1);
+      const toCase = normalizedURL.searchParams.get('toCase');
+      const validCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
please move 'validCases' const outside the the `createServer` function and use upperCase for this const